dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/Reason.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.opensource.classpath;
+
+/** The kind of linkage error against a symbol reference. */
+enum Reason {
```

Comment:
Would ErrorType be a better name here? "Reason" is very general, and doesn't even suggest anything negative.

Reply:
Updating to ErrorType: https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/598/files#diff-439f65974a3849e4bec134b2b651f64fR20